### ISSUE TITLE
Set default value of pdf2d_param_list in run_fitting.py to None

### ISSUE THREAD

**Issue by petiay** [BEAST-Fitting/beast#731]
Right now a list of parameters is hard-coded as an argument in [run_fitting.py](https://github.com/BEAST-Fitting/beast/blob/master/beast/tools/run/run_fitting.py). Generating 2D PDFs can add a significant amount of time to the fitting and by default one may not need this for every single source. 
It is probably best if a list of arguments is optionally supplied and the default for `pdf2d_param_list` is set to `None`.

**Comment by petiay** [comment]
@karllark and @cmurray-astro, do you think my proposal above makes sense? One can always pass a list of parameters for which they want to calculate the 2D PDF. 
This way by default the fitting will skip the 2D PDFs step which will significantly speed things up. 
Below is an example of the time it takes (using STScI's Science Cluster) to initially set up the 2D PDF objects currently hard-coded in run_fitting.run_fitting before the actual fitting and calculation begins. (Let alone the additional computational time for the 2D PDFs; XSEDE also takes a ton of time to set these up). 

getting bin props for Av+Rv took 0.030674457550048828 min
getting bin props for Av+f_A took 0.28903716802597046 min
getting bin props for Av+distance took 0.6248775362968445 min
getting bin props for Av+logA took 0.7510839502016703 min
getting bin props for Av+M_ini took 1.3630451838175455 min
getting bin props for Av+Z took 7.275423792997996 min
getting bin props for Rv+f_A took 7.426308413346608 min
getting bin props for Rv+distance took 7.569541033109029 min
getting bin props for Rv+logA took 7.64425200621287 min
getting bin props for Rv+M_ini took 7.89667268594106 min
getting bin props for Rv+Z took 9.978461654980977 min
getting bin props for f_A+distance took 10.061081397533417 min
getting bin props for f_A+logA took 10.14790330727895 min
getting bin props for f_A+M_ini took 10.481031727790832 min
getting bin props for f_A+Z took 13.314395809173584 min
getting bin props for distance+logA took 13.412550278504689 min
getting bin props for distance+M_ini took 13.541990387439728 min
getting bin props for distance+Z took 14.334244966506958 min
getting bin props for logA+M_ini took 14.406464159488678 min
getting bin props for logA+Z took 19.601796225706735 min
getting bin props for M_ini+Z took 19.753345076243082 min

**Comment by karllark** [comment]
I agree that we should not be doing any 2D pPDFs by default.  Makes sense as the number of possible ones is very large and the user should make a decision if they want any and which ones - especially given the computations time required.
